instagram-app/src/App.js

```diff
@@ -1,26 +1,55 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react';
 import './App.css';
+import dummyData from './dummy-data';
+import PostsContainer from './Components/PostContainer/PostsContainer';
+import SearchBar from './Components/SearchBar/SearchBarContainer';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
```

Comment:
Make sure to check your errors.

>
Compiled with warnings.

./src/App.js
  Line 23:  Expected to return a value at the end of arrow function  array-callback-return

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.

>